python/tvm/micro/model_library_format.py

```diff
@@ -22,8 +22,9 @@
 import os
 import re
 import tarfile
+import tvm
 
-from ..contrib import utils
+from ..contrib import utils, cc
```

Comment:
i think you're reverting these changes, but in case not: I think cc is a pretty surprising variable to have as a module-level thing. prefer to refer to it as contrib.cc